src/Common/tests/System/Collections/ICollection.NonGeneric.Tests.cs

```diff
@@ -208,7 +208,7 @@ public void ICollection_NonGeneric_CopyTo_TwoDimensionArray_ThrowsException(int
             }
         }
 
-        [ConditionalTheory(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNonZeroLowerBoundArraySupported))]
+        [ConditionalTheory("System.PlatformDetection, CoreFx.Private.TestUtilities!" + nameof(PlatformDetection.IsNonZeroLowerBoundArraySupported))]
```

Comment:
I just left the comment on the BuildTools PR but I think this would be better if the assemlby name was part of the type and not part of the member.